### ISSUE TITLE
Site search doesn't work

### ISSUE THREAD

**Issue by chalin** [open-telemetry/opentelemetry.io#572]
Site search doesn't seem to be working whether I use Chrome (90.0.x) or Safari (14.1): I type text and hit return (not sure if that's necessary) and nothing happens.

I'm noticing an error in the JS console:

```nocode
[Error] TypeError: undefined is not an object (evaluating 'this._logger.debug')
	patch (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:147786)
	(anonymous function) (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:59774)
	forEach
	(anonymous function) (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:59760)
	(anonymous function) (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:56762)
	(anonymous function) (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:284537)
	r (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:284837)
	(anonymous function) (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:285730)
	Global Code (main.min.d7dc1ad71467cc5386fdb301595279e0ff5ade4db320038fb19edf721b67855e.js:1:286733)
```

Clicking on the last "Global Code" frame brings me to following (although that's not the cause):

```js
 e(document).ready((function() {
            e(document).on("keypress", ".td-search-input", (function(t) {
                if (13 === t.keyCode) {
                    var r = e(this).val(),
                        n = window.location.origin + "/search/?q=" + r;
                    return document.location = n, !1
                }
            }))
        }))
    }(jQuery)
})();
```

Related: #317, #323

**Comment by austinlparker** [comment]
Thanks for the report - this was due to an update of the OTel libraries that didn't get everything.

**Comment by chalin** [comment]
Closed by #573
